### ISSUE TITLE
updating our repo structure

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/food-datasets-csv-parser#215]
**Is your feature request related to a problem? Please describe.**
Right now code inside of our repository is confusing. we keeping everything together, so it hard to follow for new people.
I did a similar changes at our generator repository. you can take a look, observe, compare changes and make similar updates.
link: https://github.com/GroceriStar/food-static-files-generator

**Comment by atherdon** [comment]
@sibasish14 you can do it as well

**Comment by sibasish14** [comment]
The structures seem to be completely different. Any suggestions about how should I proceed? 
Do you want me to move some files inside the `/package` folder? The generator repo has most of the files stored inside the `/package` folder. So we can do the same.

**Comment by atherdon** [comment]
yes, we need to move some files, that related only to module(for example -
src folder is part of the module, but projects2.0 or projects3.0 is not)
open PR, show me the changes and I'll guide you forward

>

**Comment by atherdon** [comment]
#226 please check this issue - it's part of our migration, but i'm closing this issue because main part is done

**Comment by sibasish14** [comment]
@atherdon Sorry, I'm asking again, but I haven't been assigned any more tasks. Is there any more tasks I can try?

**Comment by atherdon** [comment]
of source! thanks for buzzing me. I missed to create a task
